x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/action/PostConnectorAction.java

```diff
@@ -135,7 +139,18 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
 
         @Override
         public ActionRequestValidationException validate() {
-            return null;
+            ActionRequestValidationException validationException = null;
+
+            if (Strings.isNullOrEmpty(getIndexName())) {
+                validationException = addValidationError("index_name cannot be null or empty", validationException);
```

Comment:
I saw at some other places, that field names and values are usually between brackets.
Example: https://github.com/elastic/elasticsearch/blob/e93892c0fdfd8e0614a67368d3f746fe033ccab1/x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/operator/FileOperatorUsersStore.java#L196-L206


```suggestion
                validationException = addValidationError("[index_name] cannot be [null] or [\"\"]", validationException);
```

Not too sure about `[null] or [\"\"]`, but you get the point 😄